web_client/views/imageViewerWidget/leaflet.js

```diff
@@ -1,11 +1,19 @@
+import _ from 'underscore';
+
 import ImageViewerWidget from './base';
 
 var LeafletImageViewerWidget = ImageViewerWidget.extend({
     initialize: function (settings) {
         ImageViewerWidget.prototype.initialize.call(this, settings);
 
-        $('head').prepend(
-            $('<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css">'));
+        var cssLoaded = _.some(
+            document.styleSheets,
```

Comment:
Is webpack typically cool with referencing `document` like this, without any import?